css/theme-light.css

```diff
@@ -66,13 +66,13 @@ body[theme="light"] #tabs-list li.active {
   border-left: 10px solid #9B9B9B !important;
 }
 
-body[theme="light"] #tabs-list li:hover .close {
-  background-image: url('../images/close.svg');
-  opacity: 0.3;
+body[theme="light"] #sidebar .mdc-icon-button {
+  color: #000000;
```

Comment:
#000 should work for now and be sufficient :)

Reply:
Done.